scripts/run_tests.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+JUPYTER_PORT=32323
+IP_ADDRESS="127.0.0.1"
+OUTPUT_FILE=/tmp/output
+run_unit=0
+run_integration=0
+run_docker=0
+CONTAINER=
+
+kill_descendant_processes() {
+    local pid="$1"
+    local and_self="${2:-false}"
+    if children="$(pgrep -P "$pid")"; then
+        for child in $children; do
+            kill_descendant_processes "$child" true
+        done
+    fi
+    if [[ "$and_self" == true ]]; then
+        kill -9 "$pid"
+    fi
+}
+
+while getopts 'iuc:h' opt;
```

Comment:
this supports the same options as the python version -- run unit and/or integration tests against a local narrative or against a container.